src/AssetsPage/CopyButton/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
I've moved the copy button out into a separate component file. I put it in the AssetsPage directory, because that's where it's being used, but I imagine we could move it to some kind of utility component folder when the time comes and we find that we need this sort of button elsewhere.

Reply:
I think it could go in the `src/components` actually. Thoughts?